src/platform/target.jl

```diff
@@ -11,23 +11,25 @@ const JLL_ENV_CLANG_TARGETS_MAPPING = Dict(
     "x86_64-apple-darwin14"=>"x86_64-apple-darwin14",
     "x86_64-linux-gnu"=>"x86_64-unknown-linux-gnu",
     "x86_64-linux-musl"=>"x86_64-unknown-linux-musl",
-    "x86_64-unknown-freebsd"=>"x86_64-unknown-freebsd13.2",
+    "x86_64-unknown-freebsd13.2"=>"x86_64-unknown-freebsd13.2",
     "x86_64-w64-mingw32"=>"x86_64-w64-windows-gnu",
 )
 
 triple2target(triple::String) = get(JLL_ENV_CLANG_TARGETS_MAPPING, triple, "unknown")
 
 function __triplet(p::Platform)
     for k in keys(p.tags)
-        if k != "arch" && k != "os" && k != "libc"
+        if k != "arch" && k != "os" && k != "libc" && k != "call_abi" && k != "os_version"
```

Comment:
`call_abi` is needed for armv7l, `os_version` is needed for freebsd and sometimes darwin